pkg/controller/remotemachineset/remotemachineset_controller.go

```diff
@@ -128,6 +129,10 @@ func (r *ReconcileRemoteMachineSet) Reconcile(request reconcile.Request) (reconc
 		log.WithError(err).Error("error looking up cluster deployment")
 		return reconcile.Result{}, err
 	}
+	if cd.Annotations != nil && cd.Annotations[constants.AnnotationSyncsetPause] == "true" {
```

Comment:
Do we need to both `cd.Annotations != nil` and `cd.Annotations[constants.AnnotationSyncsetPause]`?